### PR TITLE
Add recipe for aiohttp-client-cache

### PR DESCRIPTION
**Pull request by JWCook**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by JWCook** [comment]
@conda-forge/help-python
Ready for review!

File: recipes/aiohttp-client-cache/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
```
Comment (dopplershift) [inline recipes/aiohttp-client-cache/meta.yaml:22]: I know it built successfully somehow, but this should probably list `poetry` here as well

Comment (JWCook) [reply recipes/aiohttp-client-cache/meta.yaml:22]: The current version on pypi ([v0.3.0](https://pypi.org/project/aiohttp-client-cache/0.3.0/)) was built with plain old [setup.py](https://github.com/JWCook/aiohttp-client-cache/blob/v0.3/setup.py).

I just updated the project to use poetry, but haven't made a new release yet. I was planning to update the recipe after releasing the next version. Or is there a better way to handle that?

Comment (dopplershift) [reply recipes/aiohttp-client-cache/meta.yaml:22]: Ah, I missed that--I just looked at the repo. That explains why it works. 😄 You can update the recipe when you bump the version in the feedstock.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**